src/app/db.py

```diff
@@ -19,7 +19,15 @@
     metadata,
     Column("id", Integer, primary_key=True),
     Column("username", String(50)),
+    Column("access_id", Integer, ForeignKey("access.id"), unique=True),
     Column("created_at", DateTime, default=func.now()),
+)
+
+access = Table(
+    "access",
+    metadata,
+    Column("id", Integer, primary_key=True),
+    Column("username", String(50)),
```

Comment:
If I understand correctly, this can be a user or a device so perhaps it would be best to rename it to "login", what do you think?

Reply:
Yes I was unsure about the naming. I was also afraid that login would be confusing with the "login" route but now that I think of it, we have "user" route and we don't confuse it with the entities.

Reply:
Done